docker/README.md

```diff
@@ -67,7 +67,14 @@ Jenkins Kubernetes plugin slaves:
 
 ## Build
 
-To build slave execute docker build
+S2I build is required in order to include the 'acpepted license agreement' android sdk
+Manual task - use only if changes were made to nodejs or other dependencies
 
-    docker build . -t aerogear/jenkins-android-slave
+    cd android-slave-s2i directory
+    docker build -t aerogear/jenkins-android-slave-s2i
 
+To include the android-sdk, download the required sdk, install and accept the license agreement
```

Comment:
[Optional] I think that we may include here our example installation script. It's not easy to know what would be required sdk installation etc.